docs/src/api/class-download.md

```diff
@@ -48,6 +48,15 @@ download = download_info.value
 path = download.path()
 ```
 
+```csharp
+var downloadTask = page.WaitForDownloadAsync();
+await Task.WhenAll(downloadTask, page.ClickAsync("#downloadButton"))
+    .WithTimeout(1000)
```

Comment:
- There is a natural timeout on both waitForDownload and click, why do we add another one?
- I don't think we need ConfigureAwait(false) in any of our examples and code.